.npmrc

```diff
@@ -0,0 +1,2 @@
+save-prefix=''
```

Comment:
I always forget what this is for?

Reply:
It is to avoid ^ dependencies. But you asked for that in the other PR. 

I'm a little afraid of doing that as this repo has no tests at all :-)

Reply:
Hmm, I thought that was `save-exact=true`

Reply:
That would probably also work :-)